Source/Core/DolphinQt2/Config/Mapping/MappingWindow.cpp

```diff
@@ -109,7 +109,12 @@ void MappingWindow::CreateMainLayout()
   m_main_layout = new QVBoxLayout();
   m_config_layout = new QHBoxLayout();
   m_tab_widget = new QTabWidget();
-  m_button_box = new QDialogButtonBox(QDialogButtonBox::Ok);
+  m_button_box = new QDialogButtonBox;
```

Comment:
Is it not possible to just call button()->setDefault(false) (or setAutoDefault)? It feels like some default Qt logic is being unnecessarily reimplemented.